models/member.js

```diff
@@ -38,7 +38,7 @@ const memberSchema = new mongoose.Schema({
         required: true
     },
 
-    groupChat: {
+    groupChats: {
```

Comment:
Why did you change this?

Reply:
Actually it was a typo by me, however I fixed that [here](https://github.com/hacettepeoyt/website/commit/2743589c6efd0d1df66d792906c7dee88b300139). @FeverEclipse 's branch wasn't up to date I believe.